build-farm/make-adopt-build-farm.sh

```diff
@@ -28,8 +28,18 @@ JAVA_FEATURE_VERSION=$(echo "${JAVA_TO_BUILD}" | tr -d "[:alpha:]")
 
 if [ -z "${JAVA_FEATURE_VERSION}" ]
 then
-    # THIS NEEDS TO BE UPDATED WHEN HEAD UPDATES (the latest tag that jdk/jdk contains)
-    JAVA_FEATURE_VERSION=15
+  # Use Adopt API to get the JDK Head number
+  echo "This appears to be JDK Head. Querying the Adopt API to get the JDK HEAD Number (https://api.adoptopenjdk.net/v3/info/available_releases)..."
+  JAVA_FEATURE_VERSION=$(curl https://api.adoptopenjdk.net/v3/info/available_releases | jq '.most_recent_feature_version')
```

Comment:
Is that call correct?  Is it returning 15?  There's also the tip_version API call.

Reply:
The call returns 15 as it's the latest JDK head number. What do you mean by tip_version API call?

Reply:
@M-Davies Martijn is referring to the tip_version call in the API response.

```json
{
"available_lts_releases": [
8,
11
],
"available_releases": [
8,
9,
10,
11,
12,
13,
14
],
"most_recent_feature_release": 14,
"most_recent_feature_version": 15,
"most_recent_lts": 11,
"tip_version": 15
}
```

`most_recent_feature_version` is the most recent version that we have built whereas tip_version is the definitive version of tip. In principle they will nearly always be the same but when we switch to 16 `most_recent_feature_version` will remain at 15 until we've created a nightly/release

Reply:
Ah I see. I thought they were the other way around. In that case, I'll go back and amend the other places where I'm using `most_recent_feature_version`